src/OrbitGl/ScopeTreeTest.cpp

```diff
@@ -15,17 +15,17 @@
 namespace {
 
 struct TestScope {
-  uint64_t Start() const { return start; }
-  uint64_t End() const { return end; }
-  uint64_t start;
-  uint64_t end;
+  uint64_t start() const { return start_; }
+  uint64_t end() const { return end_; }
+  uint64_t start_;
+  uint64_t end_;
 };
 
 TestScope* CreateScope(uint64_t start, uint64_t end) {
   static std::vector<std::unique_ptr<TestScope>> scope_buffer;
   auto scope = std::make_unique<TestScope>();
-  scope->start = start;
-  scope->end = end;
+  scope->start_ = start;
+  scope->end_ = end;
```

Comment:
I'm not going to split it, because it's not a nit in naming.

TextBox has two methods Start() and End(), but TimerInfo not, they have the methods start() and end() in lower case. After a short discussion with Pierric, he agreed on renaming them both as only lowercase methods. So, this test have to change from Start() to start(), and we can't have start() method and start variable, so I had to include the _ there. 

I mean, I could have started with the renaming of start_ and end_ in one commit of 4 lines, but I don't think that has any sense, what do you think?